app/javascript/components/ProductEdit/ShareTab/index.tsx

```diff
@@ -71,12 +72,14 @@ export const ShareTab = () => {
               </a>
             </header>
             {isListedOnDiscover ? (
-              <div role="status" className="success">
-                <div>{product.name} is listed on Gumroad Discover.</div>
-                <a className="close" href={discoverLink.toString()}>
-                  View
-                </a>
-              </div>
+              <Alert role="status" variant="success">
+                <div className="grid grid-cols-[1fr_auto]">
```

Comment:
Would something like `flex flex-col lg:flex-row justify-between` have the same effect? Wondering if we can avoid the `col-start-1 sm:col-start-2` below, it's nice when layouts don't depend on multiple elements' classes